css/mail.css

```diff
@@ -602,6 +602,9 @@
 .icon-sent {
 	background-image: url('../img/sent.svg');
 }
+.icon-archive {
+	background-image: url('../img/archive.svg');
```

Comment:
@zinks- Thanks for the clarification. I'm still familiarizing myself with the code, so I didn't realize that some things are integrated directly from core, like icons.css for example. It all makes sense now. :)